common/src/com/elikill58/negativity/common/protocols/AimBot.java

```diff
@@ -46,6 +46,69 @@ public AimBot() {
 		super(CheatKeys.AIM_BOT, CheatCategory.COMBAT, Materials.TNT, CheatDescription.VERIF);
 	}
 
+	// this check seems stupid, although it's a must-have
+	// every decent anti cheat needs this as it fixes most old scaffolds/killauras
+	@Check(name = "invalid", conditions = CheckConditions.SURVIVAL, description = "Checks for an invalid rotation", ignoreCancel = true)
+	public void invalidP(PacketReceiveEvent e, NegativityPlayer np) {
+		if (!e.hasPlayer())
+			return;
+		PacketType type = e.getPacket().getPacketType();
+		if (type.isFlyingPacket()) {
+			NPacketPlayInFlying flying = (NPacketPlayInFlying) e.getPacket().getPacket();
+			if (!flying.hasLook)
+				return;
+			// easiest aim check ever?
+			// and yes, it seems stupid, but many clients do not follow this
+			// especially old scaffolds will fail this check
+			final double maxP = np.isOnLadders ? 91.11d : 90d;
+			final double absolutePitch = Math.abs(flying.pitch);
+			if (absolutePitch > maxP) {
+				Negativity.alertMod(ReportType.WARNING, np.getPlayer(), this, 100, "invalid", "pitch: "
```

Comment:
Maybe you can implement the set back options. The return of `alertMod` is true if should cancel. Use `isSetBack()` with result of this and it should be fine